build/yaml/jobs/linux/binaries.yaml

```diff
@@ -35,4 +32,5 @@ jobs:
       PathToPublish: '$(Build.SourcesDirectory)/out/Linux/bin/$(Platform).$(Configuration)'
       ArtifactName: 'binaries-linux-$(Distro)-$(Configuration)'
 
-  - template: ../../steps/microbuild/cleanup.yaml
\ No newline at end of file
+  - ${{ if notIn(parameters.SignType, '', 'None') }}:
```

Comment:
>notIn [](start = 11, length = 5)

consider doing the inverse of this... check to see if the sign type is "Test" or "Real"

Reply:
Also, is this just a way to check to see if this an official build? Why not just add a build variable to Azure Devops instead? It seams like an imperfect heuristic that could just be solved by a variable.

---
In reply to: [342160850](https://github.com/microsoft/CLRInstrumentationEngine/pull/156#discussion_r342160850) [](ancestors = 342160850)

Reply:
Since MicroBuild tasks are internal, they will fail with "task not found" on public Azure DevOps projects. The only MicroBuild task we depend on is for Signing, so we can expose our PR and CI builds. MicroBuild cleanup just sends telemetry for builds that use MicroBuild tasks.

I'll refactor to use a separate variable `IsInternalProject` which would allow us to skip any task that is considered internal, not just MicroBuild.

---
In reply to: [342162437](https://github.com/microsoft/CLRInstrumentationEngine/pull/156#discussion_r342162437) [](ancestors = 342162437,342160850)